cms/pytest.ini

```diff
@@ -8,10 +8,17 @@ addopts = --nomigrations --reuse-db --durations=20 -p no:randomly --json-report
 # Enable default handling for all warnings, including those that are ignored by default;
 # but hide rate-limit warnings (because we deliberately don't throttle test user logins)
 # and field_data deprecation warnings (because fixing them requires a major low-priority refactoring)
+# ABC deprecation Warning comes from libsass
```

Comment:
Then let's suppress it only from there.

The correct format is `action:message:category:module:line`, as described in [the Python docs](https://docs.python.org/3/library/warnings.html#the-warnings-filter). In this case, we want this to be something like `ignore:Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated.*:DeprecationWarning:sass`.

Reply:
Only one that didn't work was the package resources one. it's being triggered at multiple levels of dependencies (xblock, fs, package resources, and sphinxcontrib, and maybe even setuptools). spent hours in this rabbit hole lol

Reply:
Usually, the warning displays a line of code that triggers it, so you can use grep in the virtualenv directory. It doesn't apply to this use case, though, so let's keep it as is.